exonum/src/runtime/mod.rs

```diff
@@ -170,7 +170,8 @@ use crate::{
 
 mod blockchain_data;
 mod dispatcher;
-pub(crate) mod error;
+#[doc(hidden)]
+pub mod error;
```

Comment:
I think it's possible to export only `execution_error` module in `error` without exposing the `error` module itself.